provision_scripts/install_pyart_notebooks.sh

```diff
@@ -4,9 +4,11 @@
 set -x
 # Get the Py-ART notebooks from GitHub
 cd ~
-#install git just in case
+#install git and zip/unzip just in case
 sudo apt-get install -qq git
-git clone https://github.com/openradar/pyart_short_course
-#get data
-cd pyart_short_course/data
-./get_data.sh
+sudo apt-get install -qq zip unzip
+git clone https://github.com/openradar/AMS-Short-Course-on-Open-Source-Radar-Software.git pyart_short_course
+
+# Optionally get data, add ~1 GB to the VM size
+#cd pyart_short_course/data
+#./provision_data.sh
```

Comment:
I think keeping the files out of the VM is the best idea.  There are links to the data in the notebooks and the `provision_data.sh` script is available if anyone wants to download the data.  Notebook 3-7 are rendered so that participants can browse through the materials on their own.